build.properties

```diff
@@ -28,10 +28,10 @@ install.mac.jre = ${env.OSX_JRE_HOME}
 jre.dir.name = ${env.OSX_JRE_NAME}
 
 # On Windows use PATH, on linux use LD_LIBRARY_PATH
-platform.hdf.lib = ${env.HDFVARNAME}
+platform.hdf.lib = ${env.LD_LIBRARY_PATH}
 
 # Needs to be set to /web/trunk/products/java SVN checkout directory for packageDocumentation to work
-userguide.dir = ${env.HDF-Java-Web}
+userguide.dir = /home/byrn/HDF_Projects/temp/UsersGuide
```

Comment:
Checked in build.properties

Reply:
Strange because I don't see that except here

Reply:
Oh the User Guide!